index.js

```diff
@@ -44,10 +44,12 @@ module.exports = function configParser(yaml, templateFactory) {
         .then(phaseGeneratePermutations)
         // Output in the right format
         .then(doc => ({
+            annotations: Hoek.reach(doc, 'annotations', { default: {} }),
```

Comment:
Oops, didn't realize that was a thing. Added to `beta_pipelines`. I'm not sure what `beta_rc_pipelines` is though, so I didn't modify that one.